ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonVnextDemoController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ButtonVNextDemoController: DemoController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        container.alignment = .leading
+
+        for style in ButtonVnextStyle.allCases {
+            addTitle(text: style.description)
+
+			let vnextButton = ButtonVnext(style: style, action: {
+				print("Button Vnext Clicked...")})
```

Comment:
something else than console? more like alert?

Reply:
This is addressed in the Button implementation PR that I'm creating shortly.